docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,27 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+
+## JSON and XDR Conversion Schema
+
+:::info
+
+SDF has published libraries which support the canonical and round-trippable conversion from XDR to JSON, and from JSON to XDR, without loss of information.
+
+:::
+
+The canonical JSON and XDR Schema is defined by the [stellar_xdr crate](https://docs.rs/stellar-xdr/21.1.0/stellar_xdr/index.html) and by the [@stellar/stellar-xdr-json-web npm package](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web).
+
+This conversion is visible in the Stellar CLI, Laboratory View XDR, and Hubble events table. The schema provides a round-trippable means for converting any Stellar XDR type to JSON and converting that JSON back to the identical XDR.
+
+### Key Characteristics of the JSON and XDR Conversion Schema
+
+- Round-Trippable: The JSON format allows for converting from XDR to JSON and back to XDR without loss of information.
+
+- Self-Describing: The JSON format describes the internals of the type but does not identify the type that is encoded. This is similar to XDR, which also does not identify the encoded type.
+
+- Single Valid Representation: For any given XDR, there is a single valid JSON representation, and vice-versa. This ensures reproducibility and consistency.
```

Comment:
We can't actually guarantee this for every XDR object yet. It's a WIP, but is achievable.

Reply:
Should I remove this then? It's a design goal but not a guarantee.

Reply:
I'm struggling with the best course of action here. Should we remove the "Single Valid Representation" line from here, or would it be better to add a caveat to that line?

Reply:
imo we should make sure docs are accurate as of the day they're published rather than mentioning future goals or upcoming features - those things can be announced when they happen through different channels

Reply:
that's what my gut was saying, too. Thanks for confirming

Reply:
It should be removed.